src/graphics/aurora/modelnode.h

```diff
@@ -202,6 +218,8 @@ class ModelNode {
 	Common::BoundingBox _boundBox;
 	Common::BoundingBox _absoluteBoundBox;
 
+	uint16 _nodeNumber;
```

Comment:
_nodeNumber isnt initialized in the constructor.

Reply:
Fixed